core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.ts

```diff
@@ -272,6 +272,9 @@ angular.module('oppia').directive('skillsList', [
                 // Note to developers:
                 // This callback is triggered when the Cancel button is clicked.
                 // No further action is needed.
+              }).then(function() {
+                var successToast = 'Merged Skills.';
+                AlertsService.addSuccessMessage(successToast, 1000);
```

Comment:
This seems incorrect. Why can't we add it in line 270 (right after emit)?

Reply:
It was done similarly above, so I went ahead with that. I'll move it to line 270 and re-upload the video recording.

https://github.com/oppia/oppia/blob/21b771b8f4e123a3d10bd2f876fcef9ef7f86b21/core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.ts#L214-L229

Reply:
@kevintab95  Done, PTAL